tools/build/configure.py

```diff
@@ -548,6 +563,34 @@ def build(
             },
         )
 
+        build(
+            self.build_path() / "include/gen/world_map.h",
+            [Path("src/gen/world_map.xml")],
+            "world_map",
+        )
+
+        build(
+            [
+                self.build_path() / "include/gen/move_data.h",
```

Comment:
data should be an inc.c and enums .h

Reply:
Agreed, I couldn't get configure to build with inc.c before, but its working now.